webknossos/webknossos/client/api_client/wk_api_client.py

```diff
@@ -135,6 +136,10 @@ def datastore_list(self) -> list[ApiDataStore]:
         route = "/datastores"
         return self._get_json(route, list[ApiDataStore])
 
+    def tracingstore(self) -> ApiTracingStore:
```

Comment:
We should probably decide if we want to consider tracingstore one word or two (tracing store). If it’s two, this should be called tracing_store(self), if it’s one, it should return an ApiTracingstore, not ApiTracingStore. What do you think?
The same probably applies to Datastore/DataStore

In the wk backend it’s two words